src/qibocal/protocols/characterization/allxy/allxy.py

```diff
@@ -117,7 +116,8 @@ def _acquisition(
         for qubit in qubits:
             z_proj = 2 * results[ro_pulses[qubit].serial].probability(0) - 1
             # store the results
-            data.register_qubit(qubit, z_proj, gateNumber)
+            gate = str(gates[0]) + "-" + str(gates[1])
```

Comment:
```suggestion
            gate = "-".join(gates)
```
This is a fancier way of achieving the same.